src/dialogs/more-info-dialog.html

```diff
@@ -18,9 +18,14 @@
         font-size: 14px;
       }
 
+      paper-dialog[data-domain=climate] {
+        width: 450px;
```

Comment:
Why would we only have a width specified for climate? I would consider all dialogs having a specified width except for the `camera` on (which has a `min-wdith` except when we detect that the screen is < the min width).

The width for the dialog should also be overwritten when the screen gets too small.